src/pages/dashboard/components/navBar/NavBar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from '@reach/router';
+import { Trans } from '@lingui/macro';
+
+function NavBar({ configuration, currentPath }) {
+  return (
+    <nav className="fixed pin-b pin-x bg-white md:pin-none md:pin-y md:pin-l ">
+      <ul role="tablist" className="list-reset flex px-0 md:px-3 md:flex-col lg:px-4">
+        {configuration.tabs.map((tabConfig) => {
```

Comment:
I personally prefer a more declarative way of rendering the menu, after all the main navigation will not be changing very often. thoughts?

e.g.
```
<nav>
    <Link><Trans id="menu.home" /></link>
    <Link><Trans id="menu.dashboard" /></link>
    <Link><Trans id="menu.messages" /></link>
</nav>
```

Reply:
I thought about that and... I like to have the components as little as possible that's why I decided this approach. If you think for maintability is better the way you say, we can go for it.

Somehow I think that if now a lot of people is going to touch it, it's even a better reason to have it in another file, so people don't have to deal with it.

But if people prefer it on the same component, said so!